oxidauth-usecases/src/auth/authenticate.rs

```diff
@@ -138,8 +139,17 @@ where
 
         let mut jwt_builder = Jwt::builder()
             .with_subject(user_authority.user_id)
-            .with_issuer("oxidauth".to_owned())
-            .with_not_before_from(Duration::from_secs(0));
+            .with_issuer("oxidauth".to_owned());
+
+        match authority
+            .settings
+            .jwt_nbf_offset
+        {
+            NbfOffset::Enabled { offset } => {
+                jwt_builder = jwt_builder.with_not_before_from(offset);
+            },
+            NbfOffset::Disabled => {},
+        }
```

Comment:
Check out this way of handling an enum when you only care about a subset of variants: 

https://play.rust-lang.org/?version=stable&mode=debug&edition=2024&gist=115bb6a4e3d471cdf5d81b2d21d88007